include/roboteam_utils/Time.h

```diff
@@ -41,6 +43,11 @@ class Time {
     bool operator==(const Time &other) const;
     bool operator!=(const Time &other) const;
 
+    // Returns the current date separated by the given char. Eg: "2022-12-30"
+    static std::string getDate(char separator);
+    // Returns the current time separated by the given char. Eg: "24-59-06"
```

Comment:
```suggestion
    // Returns the current time separated by the given char (HH:MM:SS). Eg: "23:59:06"
```
Not sure what kind of clock you're using, but mine doesn't go past 24 hours hahha